libs/trustchain/tests/scenarios/_template.ts

```diff
@@ -1,15 +1,14 @@
 import Transport from "@ledgerhq/hw-transport";
```

Comment:
Can you keep the convention we have in most projects for `tests` and `mocks` folders, please? By using `__tests__` and `__mocks__`

Reply:
ok but the problem is it doesn't work for the default jest config that wants to runs the files inside these even if they are outside of src. shouldn't we stay relatively jest standard?
(on top of these, they are not even jest tests)